core/src/main/java/com/airbnb/aerosolve/core/models/AbstractModel.java

```diff
@@ -46,6 +48,10 @@ abstract public float debugScoreItem(FeatureVector combinedItem,
   public double scoreProbability(float score) {
     return 1.0 / (1.0 + Math.exp(-(offset + slope * score)));
   }
+  
+  // Optional method implemented by online updatable models e.g. Spline, RBF
+  public void onlineUpdate(float grad, float learningRate, Map<String, Map<String, Double>> flatFeatures) {
```

Comment:
Throw an exception or log an error here so that people know they're calling something with no effect?